stl/inc/span

```diff
@@ -226,29 +226,38 @@ struct pointer_traits<_Span_iterator<_Ty>> {
 };
 
 // STRUCT TEMPLATE _Span_extent_type
-template <size_t _Extent>
+template <class _Ty, size_t _Extent>
 struct _Span_extent_type {
+    using pointer = _Ty*;
+
     constexpr _Span_extent_type() noexcept = default;
 
-    constexpr explicit _Span_extent_type(size_t) noexcept {}
+    constexpr explicit _Span_extent_type(const size_t, const pointer _Data) noexcept : _Mydata{_Data} {}

```

Comment:
We do not conventionally make unnamed parameters top-level `const`.
```suggestion
    constexpr explicit _Span_extent_type(size_t, const pointer _Data) noexcept : _Mydata{_Data} {}
```

Reply:
Done